### ISSUE TITLE
Machine Translations

### ISSUE THREAD

**Issue by imcgames** [Treeofsavior/EnglishTranslation#576]
Hey guys!

You might remember that there was a time when we tried adding Machine Translation to all empty lines intended to help the translators but it didn't so we deleted back most them based on your feedback. 
Some of those were left if not too much meanings were lost but there also may be some machine translated lines left unintentionally that we weren't able to remove. 
So if you come across some weird looking translations that looks like machine translations and makes absolutely no sense, please let us know about it through here or in your pull request and we'll try to have it changed to normal English asap. :sweat_smile:

**Comment by Chaosbelow** [comment]
Hmm alright but would it make it easier to add a new mark for lines that look machine translated? Say we put "m" in front of the lines. I have indicated in the past in my pull requests which lines seem machine translated and need better translation. I used to include (?) next to words lines. So if we put "m" in front of the line it might be easier for IMC and translators to fix broken lines.

**Comment by imcgames** [comment]
yup that could work too

**Comment by ensata** [comment]
Uh, yeah... About that. 

I didn't really do it seriously, but a while back I did a rough estimation of the percentage of lines that had evidence of machine translation, or were really inaccurate or had poor quality. Since very few people were willing to even look at the quest*.tsv files very seriously before me, my earliest estimates told me that 50-80% of the lines in there were not trustworthy, for various reasons.

I don't really blame you for it, but it also looked like you continued adding some machine translations after you said you wouldn't. 

I mean, really. That's 60% of the game being willingly ignored, so I'm not surprised that imc's adding rough stuff to speed things up.

**Comment by ttgmichael** [comment]
@Chaosbelow: your suggestion would work, though I would just try to translate any machine translated lines when I encounter them ^^; unless it's a really hard line to translate. So it's either a 'm' or 'x' mark.

@ensata, perhaps we need to tell more people to focus on translating quest.tsv then? It has 2276 lines (which you have taken care of several hundreds of lines, thank you :)), 60% of that is only 1000ish lines left. It's doable. (Sure they are long, but most of them have more context and are actually easier to fix grammatically than etc.tsv. xD)

**Comment by ensata** [comment]
Oh, sorry, I probably shouldn't have written it like that. 

When I say "quest*.tsv" what I mean is that the files quest.tsv, quest_jobstep.tsv, quest_lv_0100.tsv, and quest_lv_0200.tsv have a relatively low number of contributions in proportion to the other files, etc.tsv and item.tsv. In total, all the quest files account for ~21,000 lines, which is 60% of the entire game — but it's probably much higher, since these files contain lengthy dialogue and prose.

From there, I tried a few tricks to extrapolate some very rough estimates of how much text was likely to be unreliable. Since it's an extrapolation, it might only be quest.tsv, or it could be all the quest files, or the entire game — but I concluded that 50-80% of the text was not trustworthy. It might be high, but this was rough, so I could be completely wrong.

So, yeah. Not good. It's probably a big morale killer, so I didn't really want to say it... but I guess someone has to.

The best case scenario, though, is that I simply started from a really unlucky spot. If that's the case, then the first ~1300 lines in quest.tsv have signs of machine translation, and I've already looked over the first 300 now.

**Comment by ttgmichael** [comment]
I'm pretty sure there are other contributors working on the other quest files, just like those who have worked on the 15000+ lines of Etc.tsv + item.tsv. (21000+ sounds like a lot, it really isn't given our contributor size). 

Also, while there are some really funny machine translated lines out there, not all the machine translated lines are bad (some shorter lines are actually quite passable after minor grammar edits) (thank you for supplying the machine translated lines @imcgames, they are still entertaining to read xD). We shouldn't be striving to make each line "perfectly translated", just making them legible for now. There is no such thing as "perfect English", everyone has their own style of writing English (even professional translators). If you spend less time trying to pinpoint the "best" words for each line, then 21000 lines isn't that bad. Yes, it sounds like a crude method. But we can always go back with the real-time translator and rework those smaller details later.

tldr; you shouldn't be translating these lines like you are translating an epic. Strive to make the lines legible, not "perfect".

**Comment by Pomez** [comment]
Yeah the problem with quest files is that they are very hard to translate without knowing Korean. Google translate (from KR to JP to EN) can't get you far :sweat: Many contributors have worked on other files already, so I tried to look into those quest files but gave up after about 200 lines since I was never satisfied with my work (partially due to my inexperience, I guess :sob: ). Thanks to @ttgmichael, I'll try to make those lines at least understandable and leave the finishing touch to the experienced ones.

P/s: A bit off-topic but I'd like to ask: should we treat, for example, "Cleric Master" as a name or a title? Because there are times when I don't know if I should use "go to see Cleric Master" or "go to see the Cleric Master".

**Comment by ttgmichael** [comment]
@Pomez  is there a line to reference with? 

I feel it'd be awkward if someone was named "Cleric Master". So I lean towards it being a title.

**Comment by Pomez** [comment]
Ah because it seems those job masters don't have a name, they are only referred as "Cleric Master", "Wizard Master", etc. So sometimes I wonder if I should treat those titles as their names, I can feel closer with the NPCs. Maybe it's just me :sweat_smile: .However as you said, it can be awkward sometimes so I guess it's best to treat those as just mere titles.

**Comment by emailboxu** [comment]
Treat them as titles. @Pomez

**Comment by merrun** [comment]
this probably related
can someone look into SKILL_20150717_004804 (description for bow mastery) ? the translation just doesn't make sense ..its probably one of those machine translated

**Comment by B-frost** [comment]
Someone should change the How to get started page on the wiki. One of the first paragraphs talk about contributing through machine translation, and that probably confused some of the recent contributors(me included, lucky I read through the issues before doing anything else.)
